src/swap/exolix.ts

```diff
@@ -140,7 +153,7 @@ export function makeExolixPlugin(opts: EdgeCorePluginOptions): EdgeSwapPlugin {
       )
     }
 
-    const quoteAmount =
+    const nativeQuoteAmount =
```

Comment:
This is not a `native` amount. Should be renamed to `exchangeQuoteAmount` "exchange..." refers to denominations in the usual amount used by exchange. ie BTC, ETH not mETH, sats, bits.

Reply:
fixed